RHTE-LAB.adoc

```diff
@@ -0,0 +1,845 @@
+- openshift-applier (master) was installed successfully
+- extracting self to /home/lab-user/operationalizing-openshift-lab/galaxy/self
+- self (master) was installed successfully
+--------------------------------------------------------------------------------
+
+. Set and export the `LDAP_BIND_PASSWORD` environment variable:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *export LDAP_BIND_PASSWORD=xxxxxx*
+--------------------------------------------------------------------------------
++
+NOTE: The LDAP bind password is included in the authentication unit of the "Red Hat OpenShift Container Platform 4 Configuration" course in the learning management system.
+
+. Configure cluster secrets by running `openshift-applier` with the objects tagged "cluster-secrets":
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml \
+  -e include_tags=cluster-secrets -e exclude_tags=*
+--------------------------------------------------------------------------------
++
+NOTE: We must override `exclude_tags` to set cluster secrets because this variable is set in `.applier/group_vars/seed-hosts/main.yml` to exclude the `cluster-secrets` tag.
+
+. Configure Eunomia:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml \
+  -e include_tags=gitops*
+--------------------------------------------------------------------------------
+
+. Check Eunomia Configuration
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *oc get gitopsconfig cluster-config -n cluster-config -o yaml*
+apiVersion: eunomia.kohls.io/v1alpha1
+kind: GitOpsConfig
+metadata:
+  annotations:
+    gitopsconfig.eunomia.kohls.io/initialized: "true"
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"eunomia.kohls.io/v1alpha1","kind":"GitOpsConfig","metadata":{"annotations":{},"name":"cluster-config","namespace":"cluster-config"},"spec":{"resourceDeletionMode":"None","resourceHandlingMode":"None","serviceAccountRef":"eunomia-runner","templateProcessorImage":"quay.io/etsauer/eunomia-applier:latest","templateSource":{"contextDir":"","ref":"master","uri":"https://github.com/__GITOPS_ACCOUNT__/operationalizing-openshift-lab"},"triggers":[{"type":"Change"}]}}
+  creationTimestamp: "2019-09-11T19:22:09Z"
+  finalizers:
+  - eunomia-finalizer
+  generation: 3
+  name: cluster-config
+  namespace: cluster-config
+  resourceVersion: "2412063"
+  selfLink: /apis/eunomia.kohls.io/v1alpha1/namespaces/cluster-config/gitopsconfigs/cluster-config
+  uid: 73008724-d4c9-11e9-8b21-0665501aae14
+spec:
+  parameterSource:
```

Comment:
When applied correctly, there should not be a parameterSource section.

Reply:
The operator adds it.